assembly/src/parsers/nodes.rs

```diff
@@ -575,6 +580,15 @@ impl fmt::Display for Instruction {
 // HELPER FUNCTIONS
 // ================================================================================================
 
+/// Builds a hex string from a byte slice
+pub fn display_hex_bytes(f: &mut fmt::Formatter<'_>, bytes: &[u8]) -> fmt::Result {
+    write!(f, "0x")?;
+    for byte in bytes {
+        write!(f, "{byte:02x}")?;
+    }
+    Ok(())
+}
```

Comment:
The reason I have left this as a function is because we also use it here for error reporting:

https://github.com/0xPolygonMiden/miden-vm/blob/d57ae7f4e9c936afcf6f55d6775afce7731146df/assembly/src/errors.rs#L127-L130

I named it `display_hex_bytes(..)` to align with the naming of `display_push_vec(..)` below. 

https://github.com/0xPolygonMiden/miden-vm/blob/d57ae7f4e9c936afcf6f55d6775afce7731146df/assembly/src/parsers/nodes.rs#L583-L599

I'm happy to change if you still think this its best?

Reply:
Makes sense! Let's keep as is.